sql/CMakeLists.txt

```diff
@@ -566,11 +566,8 @@ SET(SQL_SHARED_SOURCES
   tztime.cc
   uniques.cc
   xa.cc
-<<<<<<< HEAD
-  ../libmysql/libmysql.cc
-=======
   join_optimizer/hypergraph.cc
->>>>>>> mysql-server/8.0
+  ../libmysql/libmysql.cc
```

Comment:
According to upstream, libmysql.cc is not necessary. Is it required for PXB?

Reply:
it is there in the previous release as well. nothing changed in this release. was as part of PXB commit 6577bdfa719234315a5bc6d4f1042143582b2ef8. should I check further?

Reply:
Just remove and see if it compiles

Reply:
it is compiling on all environment.

Reply:
Please remove this then.

Reply:
removing it. thanks for suggestion.